dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImplTest.java

```diff
@@ -57,15 +66,25 @@ public static void prepare() throws Exception{
 
         site = new SiteDataGen().nextPersisted();
     }
-
-    /*@AfterClass
-    public static void cleanup() throws DotDataException, DotSecurityException {
-
-        cleanupDebug(ESContentFactoryImplTest.class);
-    }*/
     
     final ESContentFactoryImpl instance = new ESContentFactoryImpl();
 
+    public static class TestCase {
+
+        String formatPattern;
+
+        public TestCase(final String formatPattern) {
+            this.formatPattern = formatPattern;
+        }
+    }
+
+    @DataProvider
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit 4 tests that execute tests should use the @Test annotation, JUnit 5 tests should use @Test, @RepeatedTest, @TestFactory, @TestTemplate or @ParameterizedTest](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4545506)